packages/coinjoin/src/client/CoinjoinPrison.ts

```diff
@@ -2,6 +2,7 @@ import { TypedEmitter } from '@trezor/utils';
 
 import { CoinjoinPrisonInmate, CoinjoinPrisonEvents } from '../types/client';
 import { WabiSabiProtocolErrorCode } from '../enums';
+import { ImmediateId, TimerId } from '@trezor/type-utils/libDev/src';
```

Comment:
`libDev` import :scream:

Reply:
this should be guarded by eslint, if not it was lost in latest refactor

Reply:
By this rule and wildcard https://eslint.org/docs/latest/rules/no-restricted-imports